net-misc/purritobin/purritobin-9999.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="minimalistic commandline pastebin"
+HOMEPAGE="https://bsd.ac"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/PurritoBin/PurritoBin.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/PurritoBin/PurritoBin/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+	S="${WORKDIR}/PurritoBin-${PV}"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="libuv static-libs"
+
+DEPEND="net-libs/usockets[libuv=,static-libs?]
+	libuv? ( >=dev-libs/libuv-1.35.0[static-libs?] )
+"
+BDEPEND="
+	www-apps/uwebsockets
```

Comment:
This should be in DEPEND&RDEPEND, not BDEPEND

Reply:
No, this one is correctly in BDEPEND, its a header only library, so not needed after building.

Reply:
Then not RDEPEND, but still DEPEND instead of BDEPEND.

Reply:
The same header can contain different values depending on architecture. E.g. syscall numbers or whatever. Imagine what happens if the header from /usr/include is included instead of the correct one from /usr/cross-arm/include (or whatever the cross path to sysroot would be)